skrub/_to_categorical.py

```diff
@@ -6,10 +6,21 @@ class ToCategorical(SingleColumnTransformer):
     """
     Convert a string column to Categorical dtype.
 
-    A pandas columns with dtype ``string`` or ``object`` containing strings, or
+    The main benefit is that such columns can then be recognized by
+    scikit-learn's ``HistGradientBoostingRegressor`` and
+    ``HistGradientBoostingClassifier`` with their
```

Comment:
Here, I would try to xref the models (you might need new entries in `doc/conf.py`) so that people can see what `categorical_features='from_dtype'` does